### PR TITLE
AWS Amplify Angular

### PR DESCRIPTION
**Pull request by kamilkisiela**

### CODE REVIEW

File: packages/plugins/typescript-aws-amplify-angular/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { Types, PluginValidateFn, PluginFunction } from '@graphql-codegen/plugin-helpers';
+import { visit, GraphQLSchema, concatAST, Kind, FragmentDefinitionNode, OperationDefinitionNode } from 'graphql';
+import { RawClientSideBasePluginConfig } from '@graphql-codegen/visitor-plugin-common';
+import { AwsAmplifyAngularVisitor } from './visitor';
+import { extname } from 'path';
+
+export interface AwsAmplifyAngularRawPluginConfig extends RawClientSideBasePluginConfig {}
```
Comment (dotansimha) [inline packages/plugins/typescript-aws-amplify-angular/src/index.ts]: If there is no additional configuration, you can just use `RawClientSideBasePluginConfig` directly.

File: packages/plugins/typescript-aws-amplify-angular/src/visitor.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    ];
+
+    return [baseImports, ...imports].join('\n');
+  }
+
+  public buildService() {
+    const apis = this.generatedAPIs.map(name => `${name.replace(/API$/i, '')} = ${name};`);
+
+    return `
+      @Injectable({
```
Comment (dotansimha) [inline packages/plugins/typescript-aws-amplify-angular/src/visitor.ts]: Spaces should be here?

File: packages/plugins/typescript-aws-amplify-angular/tests/plugin.spec.ts [hunk 3]
```diff
@@ -0,0 +1,221 @@
+import '@graphql-codegen/testing';
+import gql from 'graphql-tag';
```
Comment (dotansimha) [inline packages/plugins/typescript-aws-amplify-angular/tests/plugin.spec.ts]: Add `graphql-tag` to devDependencies

**Comment by dotansimha** [comment]
@kamilkisiela ?

**Comment by kamilkisiela** [comment]
@dotansimha not today, will work on it tomorrow

**Pull request closed without merging**